ui/views.py

```diff
@@ -117,13 +118,21 @@ def edit_vocabulary(request, repo_slug, vocab_slug):
 
 
 @login_required
+@permission_required_or_403(
+    # pylint: disable=protected-access
+    # the following string is "learningresources.import_course"
+    # (unless the Repository model has been moved)
+    '{}.{}'.format(
+        Repository._meta.app_label,
+        RepoPermission.import_course[0]
+    ),
+    (Repository, 'slug', 'repo_slug')
+)
```

Comment:
@giocalitri we talked briefly about this and you have a good reason, but it would be great to find a way to make this a bit more readable.